src/beta_inc.jl

```diff
@@ -371,19 +371,21 @@ function beta_inc_asymptotic_asymmetric(a::Float64, b::Float64, x::Float64, y::F
     end
     z = -nu*lnx
     if b*z == 0.0
-        return error("expansion can't be computed")
+        @debug("underflow: expansion can't be computed")
+        return w + b*z
```

Comment:
Since all variables are of type `Float64`, it seems we could also just write
```suggestion
        return w
```

Reply:
Ah yes. Old promotion habits. I've fixed it.